### PR TITLE
Add preloaders to ScholarX 2020 & 2019 mentee's section

### LINKED ISSUE #1094: Add preloaders to ScholarX 2020 & 2019 mentee's section
**Is your feature request related to a problem? Please describe.**
On the ScholarX 2020,2019 pages, the mentee's section is taking some time to load the data

**Describe the solution you'd like**
Add preloaders to ScholarX pages to avoid that

* Take a look at the PR for the IRC page research intern's section  (https://github.com/sef-global/sef-site/pull/1092)
* Already the CSS is created for the preloader
* Add the related content to the HTML files

### PR DESCRIPTION
**Pull request by fustroli**
## Purpose

The purpose of this PR is to fix #1094 

## Goals

Add skeleton preloader to ScholarX 2020 & 2019 mentee's section

## Approach

Inserted skeleton HTML code, to the appropriate sections

### Screenshots

Screenshot from 2021-08-23 09-41-18
  

### Preview Link

https://pr-1097-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
 
#1092 

## Test environment
 
Linux, Brave, Chrome

### CODE REVIEW

**Review by janethavi — DISMISSED** [review]
Good work!

File: scholarx/archive/2019/index.html [hunk 1]
```diff
@@ -78,7 +78,60 @@ <h1 class="display-3 text-center mb-5">Mentors</h1>
...
         <div class="justify-content-center">
             <!--Mentees table goes here-->
-            <div id="teamMentees"></div>
+            <table class="table table-striped">
+                <thead>
+                <tr>
+                    <th scope="col"></th>
+                    <th scope="col">Name</th>
+                    <th scope="col">University</th>
+                    <th scope="col">View Blog Article</th>
```
Comment (Piumal1999) [inline scholarx/archive/2019/index.html]: 'View blog article' option is not there in the ScholarX 2019 table. Remove it and add 'Mentor' as that table header.

Also it creates another header row after loading. 
image

You can fix that by removing the duplicate parts in the mustache template.

```
<table class="table table-striped">
        <thead>
          <tr>
            <th scope="col"></th>
            <th scope="col">Name</th>
            <th scope="col">University</th>
            <th scope="col">Mentor</th>
          </tr>
        </thead>
```

**Review by Piumal1999 — CHANGES_REQUESTED** [review]

File: scholarx/archive/2019/index.html (hunk 1 above)
Comment (fustroli) [reply scholarx/archive/2019/index.html]: Requested changes are done.
Screenshot from 2021-10-08 09-32-27

File: scholarx/archive/2020/index.html [hunk 2]
```diff
@@ -129,16 +129,58 @@ <h1 class="display-3 text-center mb-5">Mentees</h1>
...
                 <thead>
-                  <tr>
+                <tr>
                     <th scope="col"></th>
                     <th scope="col">Name</th>
                     <th scope="col">University</th>
-                    <th scope="col">Mentor</th>
-                    
-                  </tr>
+                    <th scope="col">View Blog Article</th>
```
Comment (Piumal1999) [inline scholarx/archive/2020/index.html]: Here also, it should be 'Mentor'. Not 'View Blog Article'

File: scholarx/archive/2019/index.html [hunk 3]
```diff
@@ -137,14 +190,6 @@ <h5 class="title">
 
 <script type="text/html" id="templateMentees">
```
Comment (Piumal1999) [inline scholarx/archive/2019/index.html:191]: Let's update the mentee template like this:

```
<script type="text/html" id="templateMentees">
    {{#data}}
        <tr onclick="window.location='{{linkedin}}';">
            <th scope="row">
                <img src="https://res.cloudinary.com/dsxobn1ln/image/upload/c_scale,h_48,w_48/{{image}}" class="avatar">
            </th>
                <td>{{name}}</td>
                <td>{{university}}</td>
                <td>{{mentor}}</td>
        </tr>
    {{/data}}     
</script>
```

Comment (Piumal1999) [reply scholarx/archive/2019/index.html:191]: That's how it is done on the scholarx/archive/2020/index.html page

**Review by Piumal1999 — CHANGES_REQUESTED** [review]
Hi @VWRoli, I added some comments and suggestions. Could you please check those and update the pull request? Sorry for bothering you

File: scholarx/archive/2019/index.html (hunk 3 above)
Comment (fustroli) [reply scholarx/archive/2019/index.html:191]: > Hi @VWRoli, I added some comments and suggestions. Could you please check those and update the pull request? Sorry for bothering you

No worries @Piumal1999.  Fixed the things you asked.

File: scholarx/archive/2019/index.html (hunk 1 above)
Comment (Piumal1999) [reply scholarx/archive/2019/index.html]: Well done! Thank you @VWRoli

**Review by Piumal1999 — APPROVED** [review]

**Pull request merged**